cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoContent.tsx

```diff
@@ -78,27 +85,54 @@ const EdgeInfoContent: FC<{ selectedEdge: NonNullable<SelectedEdge> }> = ({ sele
                 <>
                     {Object.entries(sections).map((section, index) => {
                         const Section = section[1];
+                        const sectionKeyLabel = section[0] as keyof typeof EdgeSections;
+                        const expandedSections = edgeInfoState.expandedSections;
+                        const isExpandedSection = expandedSections[sectionKeyLabel];
 
                         const sendOnChange =
                             EdgeCompositionRelationships.includes(selectedEdge.name) && section[0] === 'composition';
 
+                        const setExpandedPanelSectionsParam = () => {
+                            setExploreParams({
+                                expandedPanelSections: [sectionKeyLabel],
+                                ...(sectionKeyLabel === 'composition'
+                                    ? { searchType: 'composition' }
+                                    : { searchType: null }),
+                                ...(sectionKeyLabel === 'composition' && { relationshipQueryItemId: selectedItem }),
```

Comment:
Is there an edge case im missing that requires 2 separate conditionals instead of one? By 1 conditional I mean:

```suggestion
                                ...(sectionKeyLabel === 'composition'
                                    ? { searchType: 'composition', relationshipQueryItemId: selectedItem }
                                    : { searchType: null }),
```

Reply:
good catch, I tried it but it was only one param for some reason, let me give it another try

Reply:
Nice!! Im guessing there has been a lot of back and forth in these components so solid chance it was needed at one point!

Thanks for the update!!